src/main/clojure/clara/rules/engine.cljc

```diff
@@ -118,6 +118,14 @@
   ;; Right-activate the node with items reduced in the above pre-reduce step.
   (right-activate-reduced [node join-bindings reduced  memory transport listener]))
 
+(defprotocol IAccumInspect
+  ;; Takes a token that was previously propagated from the node
```

Comment:
Small thing, but `defprotocol` supports doc strings on both the protocol and each of the functions defined along with it.